core/trino-main/src/main/java/io/trino/operator/BigintGroupByHash.java

```diff
@@ -51,6 +56,12 @@
     private static final float FILL_RATIO = 0.75f;
     private static final List<Type> TYPES = ImmutableList.of(BIGINT);
     private static final List<Type> TYPES_WITH_RAW_HASH = ImmutableList.of(BIGINT, BIGINT);
+    private static final int BATCH_SIZE = 256;
+    /**
+     * Above that number of groups it is faster to use the batched way of execution.
```

Comment:
Maybe describe why it's worse to use batching for small number of groups (e.g. worse inlining, array allocation, etc). Why group count rather than page size?